Sources/DelegateProxy/ObjcDelegateProxy/ObjcDelegateProxy+Selector.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ObjcDelegateProxy+Selector.swift
+//  CombineCocoa
+//
+//  Created by Joan Disho on 25/09/2019.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+import Foundation
+
+extension ObjcDelegateProxy {
+    @objc(canRespondToSelector:)
+    public func canRespond(to selector: Selector) -> Bool {
+        selectors.contains(where: { $0 as? Selector == selector })
+    }
+
+    @objc(selectors:encodedReturnType:)
+    public static func selectorsOfClass(_ c: AnyClass, encodedReturnType: String) -> NSSet {
+        NSSet(set: selectors(ofClass: c, encodedReturnType: encodedReturnType))
+    }
+
+    private static func selectors(ofClass c: AnyClass, encodedReturnType: String) -> Set<Selector> {
+        var protocolsCount: UInt32 = 0
+        guard let protocolPointer = class_copyProtocolList(c, &protocolsCount) else { return .init() }
```

Comment:
You have a memory leak here, forgot to `defer free(protocolPointer)`

Reply:
Yup that seems correct. @jdisho can you look into this?